nextcord/enums.py

```diff
@@ -575,6 +575,7 @@ class ApplicationCommandOptionType(Enum):
     role = 8
     mentionable = 9
     number = 10  # A double, AKA floating point.
+    attachment = 11
```

Comment:
This may just be mobile. But is there a reason this is indented and the rest aren't?